project.clj

```diff
@@ -44,6 +44,7 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/i18n]
                  [nrepl/nrepl]
+                 [cider/cider-nrepl "0.21.0"]
```

Comment:
Should this version number live in clj-parent?

Reply:
most likely. non-trivial dance to get it there, tho.